locations/spiders/currys_pc_word.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+import scrapy
+from locations.items import GeojsonPointItem
+
+countries = ['england', 'uk', 'ireland', 'scotland']
+
+formdata = {'sFormName': 'storeFinder',
+                'storeFind': 'true',
+                'subaction': 'stores_map',
+                'sStoreKeywordHidden':'england', # Previous search. The choice of england was arbitrary 
+                'sStoreGeo': '',          # Appears to not be used
+                'sStoreKeyword': ''}      # Search term 
+
+headers = {'Referer': 'https://www.pcworld.co.uk/gbuk/s/find-a-store.html', 'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64…) Gecko/20100101 Firefox/57.0'}
+
+class CurrysPcWorldSpider(scrapy.Spider):
+    counter = 0 # Used for refs as the stores do not have store numbers
+    name = "currys"
+    allowed_domains = ["https://www.currys.co.uk"] # Change if using PC World 
+    # start_urls = ('https://www.currys.co.uk/gbuk/s/find-a-store.html')
+    search_url = 'https://www.pcworld.co.uk/gbuk/s/find-a-store.html'
+    url = 'https://www.pcworld.co.uk/gbuk/s/3100/find-a-store.html' # response.xpath('//li//@data-location').extract()
```

Comment:
This URL doesn't seem to be used anywhere.

Reply:
I will delete the unused URL.